From randy.dunlap@oracle.com  Thu Feb 25 15:59:04 2010
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Thu, 25 Feb 2010 14:41:22 -0800
Subject: Staging/pohmelfs: fix write_inode parameter warning
To: Stephen Rothwell <sfr@canb.auug.org.au>, devel@driverdev.osuosl.org
Message-ID: <20100225144122.d9ba8810.randy.dunlap@oracle.com>


From: Randy Dunlap <randy.dunlap@oracle.com>

sparse warns that:
drivers/staging/pohmelfs/inode.c:1797: warning: initialization from incompatible pointer type

so fix pohmelfs_write_inode() to have the expected parameters.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Evgeniy Polyakov <zbr@ioremap.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/pohmelfs/inode.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/drivers/staging/pohmelfs/inode.c
+++ b/drivers/staging/pohmelfs/inode.c
@@ -396,7 +396,8 @@ int pohmelfs_remove_child(struct pohmelf
 /*
  * Writeback for given inode.
  */
-static int pohmelfs_write_inode(struct inode *inode, int sync)
+static int pohmelfs_write_inode(struct inode *inode,
+				struct writeback_control *wbc)
 {
 	struct pohmelfs_inode *pi = POHMELFS_I(inode);
 
